docs/guides/getting_started/intro.md

```diff
@@ -136,69 +146,68 @@ handler.
 
 >[!NOTE]
 Connection logic is incomplete as of the current build. Events will
-soon be added to indicate when the client's state is ready for use;
-(rewrite this section when possible)
+soon be added to indicate when the client's state is ready for use
+(rewrite this section when possible).
 
 Finally, we will want to block the async main method from returning
-until after the application is exited. To do this, we can await an
-infinite delay, or any other blocking method, such as reading from
+until after the application has exited. To do this, we can await an
+infinite delay or any other blocking method, such as reading from
 the console.
 
 The following lines can now be added:
 
 [!code-csharp[Create client](samples/intro/client.cs)]
 
-At this point, feel free to start your program and see your bot come
+At this point, feel free to start your program and see your bot comes
```

Comment:
The original wording was correct.
As an alternative, maybe try "If all goes well, upon starting your program your bot should appear online."

Reply:
Future tense, you'd use "come." As in, "your bot will come online," as opposed to "your bot will comes online."

Reply:
Ah, alright. Will fix.